scripts/cli/actions/generate.ts

```diff
@@ -0,0 +1,48 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import selectPrompt from 'select-prompt'
```

Comment:
`select-prompt` seems to be depracated / archived. 

Let's change to `https://github.com/enquirer/enquirer` as it is already used in the project 😄 

for example here:
`import { prompt } from 'enquirer'`
...
```
const generatePrompts = data.map(({ title, value }) => ({ name: title, value }))

export const generate = async () => {
  const { generatorName } = (await prompt({
    choices: generatePrompts,
    message: 'What do you want to generate?',
    name: 'generatorName',
    type: 'select',
  })) as unknown as { generatorName: string }

  const command = data.find((item) => item.value === generatorName)?.command
  if (command) {
    command()
  } else {
    console.log('There was some issue while running the script ', generatorName)
  }
}
```

Reply:
Same in other places where `select-prompt` is used